docs/modules/ROOT/pages/explanations/gcp/name_lengths.adoc

```diff
@@ -0,0 +1,85 @@
+= Names and name length on Google Cloud Platform
+:stem:
+
+[abstract]
+.Abstract
+Node names are limited to a length of 63.
```

Comment:
Maybe add a link to here? https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names